### ISSUE TITLE
Singleplayer world crashing after a few minutes of playing

### ISSUE THREAD

**Issue by FindingMeme** [IrisShaders/Iris#1656]
### Minecraft Version

Minecraft 1.18.2

### Iris Version

iris-mc1.18.2-1.3.1.jar

### Sodium Version

sodium-fabric-mc1.18.2-0.4.1+build.15.jar

### Operating System

Windows 10

### What is your GPU?

Nvidia GEForce RTX 2060 Super

### Java Version

Curseforge Default (I think its Java 16 for 1.17)

### Reproduction Steps

1. Open Singleplayer world
2. Do anything for a few minutes
3. Game Crashes without crash report

### Crash Report file and latest.log

```shell
https://paste.gg/p/anonymous/946fac292ba24e6ba23ee65f5cabc34b
```

### Additional context

Just know this is on Valhelsia Enhanced Vanilla Modpack Version 1.4.2b.

No issues at all when playing on a multiplayer server, However when I went to singleplayer this started happening.

Had someone who works on MCEternal help me find out whats happening, and they referred me here after not knowing what is causing issue. Seemed to kinda fix itself once just by holding a stack of torches, however next launch this doesn't work again.

**Comment by IMS212** [comment]
Should be fixed in 1.4. https://github.com/IrisShaders/Iris/commit/b480ed6cabee6744d070046d735c29a49aa2456e

**Comment by IMS212** [comment]
Fixed.
